app/com/linkedin/drelephant/tunin/APIFitnessComputeUtil.java

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright 2016 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.linkedin.drelephant.tunin;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.net.URLEncoder;
+import java.util.List;
+import java.util.Random;
+
+import models.JobDefinition;
+import models.JobExecution;
+import models.TuningJobDefinition;
+import models.TuningJobExecution;
+import models.TuningJobExecution.ParamSetStatus;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.security.authentication.client.AuthenticatedURL;
+import org.apache.hadoop.security.authentication.client.AuthenticationException;
+import org.apache.log4j.Logger;
+import org.codehaus.jackson.JsonNode;
+import org.codehaus.jackson.map.ObjectMapper;
+
+import com.linkedin.drelephant.ElephantContext;
+import com.linkedin.drelephant.math.Statistics;
+
+import play.libs.Json;
+
+
+/**
+ * This is another version of FitnessComputeUtil which uses Dr Elephant API to compute the fitness.
+ * This can be used during test when Dr Elephant DB is not getting data properly to compute the fitness
+ */
+public class APIFitnessComputeUtil extends FitnessComputeUtil {
```

Comment:
Where is this used ?

Reply:
This was done for testing as local server was taking too much time to update job details in Dr Elephant DB. Can be used for testing in local dev box in future.

Reply:
Can you please include instructions in the comments or the README file on how this can be used for Testing?